roles/core/log_server/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+
```

Comment:
```suggestion
```

Reply:
fixed... btw, you can squash my commits while merging if needed.